doc/model.rst

```diff
@@ -0,0 +1,683 @@
+*****
+Model
+*****
+========
+Overview
+========
+SmartSim Models are an abstract representation for compiled applications,
+scripts, and general computational tasks. Models can be launched with
+other SmartSim entities and ML infrastructure to build AI-enabled workflows.
+SmartSim enables AI integration in ``Model`` workflows by allowing users to dynamically load
+TensorFlow (TF) functions, TorchScript, and various machine learning models
+(TF, TF-lite, PyTorch, or ONNX) into the database at runtime.
+Models are flexible enough to support many different applications, however,
+to be used with SmartSim clients (SmartRedis) the application must be written
+in Python, C, C++, or Fortran.
+
+When initializing a ``Model``, users
+provide executable simulation code to the Models run settings as well as
+execution instructions with regard to the workload
+manager (e.g. Slurm) and available compute resources. Users can specify
+and assign values to parameters used within the simulation via the `params`
+initialization argument. Parameters supplied in the `params` argument can either be
+written into supplied configuration files or be assigned within configuration files
+for use within the simulation at runtime. This functionality is supported by the
```

Comment:
This is confusing: `Parameters supplied in the ``params`` argument can either be
written into supplied configuration files or be assigned within configuration files`

...boils down to:

"can be written to configuration files or assigned to configuration files"

which sounds like _"you have two options. A or A"_

I think we're trying to say "you can supply parameters in params OR use a configuration file?"

Reply:
I think the section below (Initialize Model Types) actually duplicates this information and is a better location for the details than the overview?

Reply:
I removed this info! I agree the overview is an overview and probably shouldnt contain info on init arguments or I mean init arguments specifically